stack-easybib/templates/default/easybib.com.conf.erb

```diff
@@ -0,0 +1,103 @@
+<%= render_upstream(@php_upstream, @upstream_name) -%>
+
+<% 
+
+alias_config = NginxHelpers::Helpers::merge_alias_config_from_node(node, @app_dir)
```

Comment:
@fh @baccenfutter it's not that much:
https://github.com/till/easybib-cookbooks/blob/64da323a2b56291a913ae9f4e7868d0dd9af1194/nginx-app/spec/fixtures/recipes/silex-template.rb#L7